torch/ao/quantization/observer.py

```diff
@@ -1492,10 +1492,14 @@ def load_observer_state_dict(mod, obs_dict):
 
 # TODO(future PR): remove these defaults and enforce activation functions
 # to explicitly specify their output range
-default_symmetric_fixed_qparams_observer = FixedQParamsObserver.with_args(
+default_symmetric_fixed_qparams_range_neg1to1_observer = FixedQParamsObserver.with_args(
     scale=2.0 / 256.0, zero_point=128, dtype=torch.quint8, quant_min=0, quant_max=255)
-default_affine_fixed_qparams_observer = FixedQParamsObserver.with_args(
+default_affine_fixed_qparams_range_0to1_observer = FixedQParamsObserver.with_args(
```

Comment:
should we remove `affine` from the name?

Reply:
does range 0 to 1 imply it's affine quant? if not, and we remove affine, then I think the name could be ambiguous?

Reply:
usually in this codebase `affine` means "quantized values are not forced to be centered around zero", which is unrelated to how it is used in this variable name.  It would be cleaner just to delete `affine` from the variable name.